### DISCUSSION TITLE
New widgets don't allow multiple habits?

### DISCUSSION THREAD

**Discussion by Ballardst** [iSoron/uhabits#1992] [Help & FAQ]
The widgets I currently have show multiple habits - I pick which ones I want the widget to show and then I can swipe on it to switch in between them. Whenever I try to set up a new widget though I'm not giving that option and I have to pick a specific widget for each habit. Did something change? I find a particularly odd that the old widgets still work right I just can't set up new ones.

Android 13; Pixel 6

**Comment by ahcheing** [comment]
Something *did* change. The dev removed this feature because they where unable to make the feature "stable across multiple phones". But imo this is terrible, because currently the only way to *edit* stacked widgets is to make new ones. https://github.com/iSoron/uhabits/commit/1df9cc7664c9f896ddedff8ecfd5f2f35afb0d1f

Im debating regressing to previous version of the app but am worried I will lose data or that things will break. I really, *really* need stacked widget support.

**Comment by Ballardst** [comment]
Yep that fixed it (using an older version). Thanks!

**Reply by CODEsSAMI** [reply]
i still face the problem. which version are you using actually to get solution of it?
